lib/v2/nicesss/utils.js

```diff
@@ -0,0 +1,57 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+function parseContent(content) {
+    // 处理Content
+    const $ = cheerio.load(content);
+    $('noscript').remove(); // 删除所有的<noscript>元素
+    $('img').each(function () {
+        // 更新所有<img>元素的src属性为data-srcset的值
+        const srcset = $(this).attr('data-srcset');
+        if (srcset) {
+            $(this).attr('src', srcset);
+        }
+    });
+    return $.html();
+}
+
+async function processCategory(ctx, id) {
+    // 获取Category
+    return await ctx.cache.tryGet(`nicesss_categories:${id}`, async () => {
+        const { data: response } = await got(`https://www.nicesss.com/wp-json/wp/v2/categories/${id}`);
+        return response;
+    });
+}
+
+async function processTag(ctx, id) {
+    // 获取Tag
+    return await ctx.cache.tryGet(`nicesss_tags:${id}`, async () => {
+        const { data: response } = await got(`https://www.nicesss.com/wp-json/wp/v2/tags/${id}`);
+        return response;
+    });
+}
+
+async function processItems(ctx, response) {
+    let items = response.map(async (item) => ({
+        title: item.title.rendered,
+        link: item.link,
+        description: parseContent(item.content.rendered),
+        author: await processCategory(ctx, item.categories[0]).then((category) => category.name),
+        category: item.tags,
```

Comment:
`tags` here are tag id instead of mapped tag.

As mentioned before in https://github.com/DIYgod/RSSHub/pull/14364#pullrequestreview-1862510879, you can include the `_embed` search parameter to obtain mapped categories, tags and author from `_embedded` properties